sdk/eventhubs/azure-messaging-eventhubs/test/eventhubs-stress-test/README.md

```diff
@@ -0,0 +1,29 @@
+# Stress test prototype 
+This is work in progress. It's a prototype of how a stress test would look. This PR is to park the work in progress while being dealing with higher priority issues.
```

Comment:
```suggestion
This is work in progress. It's a prototype of how a stress test would look. 
```

Reply:
FWIW, This is literally a cut&paste from the stress test in azure core.